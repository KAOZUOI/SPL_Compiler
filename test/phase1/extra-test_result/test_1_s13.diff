1,10c1,3
< Error type A at Line 1: unknown lexeme 
< Error type B at Line 1: Missing semicolon ';' LOC2
< Error type A at Line 2: unknown lexeme 
< Error type B at Line 2: Missing semicolon ';' LOC2
< Error type B at Line 3: Missing semicolon ';' LOC2
< Error type B at Line 1: Missing semicolon ';' LOC2
< Error type A at Line 3: unknown lexeme 
< Error type A at Line 4: unknown lexeme 
< Error type A at Line 5: unknown lexeme 
< Error type A at Line 6: unknown lexeme 
---
> Error type B at Line 3: Missing closing braces ']'
> Error type B at Line 5: Missing closing parenthesis ')'
> Error type B at Line 1: Missing closing curly bracket '}'
