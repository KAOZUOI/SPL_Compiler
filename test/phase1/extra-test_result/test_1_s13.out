Error type A at Line 1: unknown lexeme 
Error type B at Line 1: Missing semicolon ';' LOC2
Error type A at Line 2: unknown lexeme 
Error type B at Line 2: Missing semicolon ';' LOC2
Error type B at Line 3: Missing semicolon ';' LOC2
Error type B at Line 1: Missing semicolon ';' LOC2
Error type A at Line 3: unknown lexeme 
Error type A at Line 4: unknown lexeme 
Error type A at Line 5: unknown lexeme 
Error type A at Line 6: unknown lexeme 
